scenario/BulkerScenario.ts

```diff
@@ -10,15 +10,16 @@ scenario(
     filter: async (ctx) => await isBulkerSupported(ctx) && !matchesDeployment(ctx, [{deployment: 'weth'}, {network: 'mumbai'}]),
     supplyCaps: {
       $asset0: 3000,
+      $asset2: 1,
```

Comment:
This scenario assumes that the wrapped native token collateral asset has a supply cap of at least `.01` units.

In the Polygon deploy, that asset is `$asset2`; somewhat awkward to assume which asset is the wrapper

Reply:
Yeah hmm, should we try to add a marker or something (e.g. in NativeTokenConstraint)?

Reply:
like something that would let you specify a `supplyCaps` value for `$nativeTokenWrapper`, and the constraint would find which asset is the wrapper?

Reply:
Yeah, maybe just `$wnative` is an alias?